L1Trigger/L1THGCal/plugins/be_algorithms/HGCClusterAlgo.cc

```diff
@@ -52,6 +56,16 @@ class HGCClusterAlgo : public Algorithm<FECODEC>
                     << "'. Using nearest neighbor NNC2d instead.\n";
                 clusteringAlgorithmType_ = NNC2d;
             }
+            std::string typeMulticluster(conf.getParameterSet("C3d_parameters").getParameter<std::string>("type_multicluster"));
+            if(typeMulticluster=="dRC3d"){
+                multiclusteringAlgoType_ = dRC3d;
+            }else if(typeMulticluster=="DBSCANC3d"){
+                multiclusteringAlgoType_ = DBSCANC3d;
+            }else {
+                edm::LogWarning("ParameterError") << "Unknown Multiclustering type '" << typeMulticluster
+                    << "'. Using DBSCAN instead.\n";
+                multiclusteringAlgoType_ = DBSCANC3d;
```

Comment:
This algo should not be used as default for the moment since no performance studies have been shown yet.